Readme.md

```diff
@@ -71,6 +71,33 @@ import { svelteNative } from 'svelte-native'
 svelteNative(App, {msg: "Hi from launcher"});
 ```
 
+## Examples
+
+### Svelte Native HackerNews
+
+Simple HackerNews client in Svelte Native.
+
+See https://github.com/halfnelson/svelte-native-hackernews for the repo.
+
+(https://raw.githubusercontent.com/halfnelson/svelte-native-hackernews/master/nativescript-svelte-hn.gif)
```

Comment:
I think it would be better to show these images inline, by using the following syntax.

```suggestion

![HackerNews Example Image](https://raw.githubusercontent.com/halfnelson/svelte-native-hackernews/master/nativescript-svelte-hn.gif)
```

Reply:
Ooh, yes of course! Thank you for the catch. I changed them over to be inline gifs, rather than just links.